org/pr/ios-macos.ts

```diff
@@ -18,10 +18,7 @@ export default async () => {
     const podfileContents = await danger.github.utils.fileContents("Podfile");
     const matches = podfileContents.match(/^[^#]*:commit/gm);
```

Comment:
I think this might give a false negative for gutenberg because of this line:
https://github.com/wordpress-mobile/WordPress-iOS/blob/342f9811440d740767136a42f0beac624051f309/Podfile#L133

I'm not really knowledgable with Ruby so I'm not sure how to keep the functionality same but eliminate the false negative by modifying the `Podfile`. Maybe we can modify the regex in a way that doesn't catch this line but other cases.

Reply:
Good catch @ceyhun! Thank you!
I've updated the regex so that it won't catch `[:commit]` and I also added a few tests to make sure it worked.